src/main/scala/crime.scala

```diff
@@ -0,0 +1,56 @@
+import scala.io.Source
+import java.io.File
+
+object start {
+  def getListOfFiles(dir: String): List[File] = {
+    val listFiles = new File(dir).listFiles
+    val res = listFiles.filter(_.isFile).toList.filter(_.getName.endsWith("csv"))
+    res
+  }
+
+  def getColls(row: String): List[String] = {
+    row.split(",").zipWithIndex.filter { case (v, index) => (List(0, 4, 5, 9) contains index) && v.nonEmpty }.drop(1).map(_._1).map(_.trim).toList
+  }
+
+  def CSVSource(source: Source): List[List[String]] = {
+    val rows = source.getLines().toList.drop(1).map(getColls)
+    rows.filter(_.length == 3)
+  }
```

Comment:
usually it is more readable and maintainable to have some data structure like case class instead of working with lists